common/service/config/kafkaConfig.go

```diff
@@ -100,3 +101,8 @@ func (k *KafkaConfig) GetBrokersForKafkaCluster(kafkaCluster string) []string {
 func (k *KafkaConfig) GetTopicsForApplication(app string) TopicList {
 	return k.Applications[app]
 }
+
+// GetKafkaVersionForApplication gets kafka version from application
+func (k *KafkaConfig) GetKafkaVersionForApplication() string {
```

Comment:
I think we should remove this method. 
The pattern of "ForApplication" is to exact config from `k.Application` 
For this version, just get it directly